TheraLang.DAL/IttmmDbContext.cs

```diff
@@ -6,56 +6,56 @@
 
 namespace TheraLang.DAL
 {
-    public class IttmmDbContext : DbContext
+    public sealed class IttmmDbContext : DbContext
     {
         public IttmmDbContext(DbContextOptions options) : base(options)
         {
             //Database.Migrate();
         }
 
         #region UTTMM_Entities
-        public virtual DbSet<Project> Projects { get; set; }
-        public virtual DbSet<ProjectType> Types { get; set; }
-        public virtual DbSet<ResourceCategory> ResourceCategories { get; set; }
-        public virtual DbSet<Resource> Resources { get; set; }
-        public virtual DbSet<ResourceProject> ResourceProject { get; set; }
-        public virtual DbSet<ProjectParticipation> ProjectParticipations { get; set; }
-        public virtual DbSet<Donation> Donations { get; set; }
-        public virtual DbSet<Society> Societies { get; set; }
-        public virtual DbSet<ResourceAttachment> ResourceAttachments { get; set; }
+        public DbSet<Project> Projects { get; set; }
```

Comment:
could you leave all virtual modifiers here and in all entities?

Reply:
won't be using lazy loading, so don't think we need virtual

Reply:
@mod1fy  @aaandrianaaa 
we are already using lazy loading,since we haven't turned off it here https://github.com/ivanko-yuz/TheraLang/blob/635f2fa49d83ecff98d850272eb08f876d1ea72c/TheraLang.DAL/IttmmDbContext.cs#L13
lazy loading is inbuild in EntityFramework, by removing **virtual** we kind of turning it off,

Reply:
https://www.tutorialspoint.com/entity_framework/entity_framework_lazy_loading.htm